### DISCUSSION TITLE
Forgot password

### DISCUSSION THREAD

**Discussion by ebhdoc** [freeplane/freeplane#2123] [Help]
I encrypted my mindmaps and haven't reviewed them for some time and now cannot access them because I don't remember the password. Is there any way to resolve this?

Many thanks!

**Comment by macmarrum** [comment]
Links to other discussions on the topic:

- https://github.com/freeplane/freeplane/discussions/1807#discussioncomment-9271272
- https://github.com/freeplane/freeplane/discussions/2127
